CompanionApp/Companion/App.xaml.cs

```diff
@@ -1,41 +1,49 @@
-using Xamarin.Forms;
-using UXDivers.Grial;
-using Xamarin.Essentials;
+using Companion.HttpClients;
 using System;
+using Xamarin.Essentials;
+using Xamarin.Forms;
 
 
 namespace Companion
 {
     public partial class App : Application
     {
+        public static ApiClient ApiClient { get; private set; }
+
+        public static bool IsDevelopment { get; set; } = true;
```

Comment:
Have you seen any sort of configuration file in the project or something to manage environments? If not it would probably be helpful to work toward it.